tools/cli/go-whisk/whisk/action.go

```diff
@@ -51,6 +51,7 @@ type Exec struct {
     Init        string      `json:"init,omitempty"`
     Main        string      `json:"main,omitempty"`
     Components  []string    `json:"components,omitempty"`    // List of fully qualified actions
+    Binary      *bool       `json:"binary,omitempty"`
```

Comment:
grr would rather see less not more :)

Reply:
Turns out we can just use the struct to organize data in the code making it easier to deal with and print the entire payload from the HTTP response instead.